src/vs/workbench/contrib/issue/electron-sandbox/issueFormService.ts

```diff
@@ -0,0 +1,212 @@
+import { AuxiliaryWindowMode, IAuxiliaryWindowService } from 'vs/workbench/services/auxiliaryWindow/browser/auxiliaryWindowService';
+
+export class IssueFormService2 implements IIssueFormService {
+
+	private configuration: IssueReporterWindowConfiguration | undefined;
+
+	private extensionIdentifierSet: ExtensionIdentifierSet = new ExtensionIdentifierSet();
+
+	declare readonly _serviceBrand: undefined;
+
+	private currentData: IssueReporterData | undefined;
+
+	private issueReporterWindow: Window | null = null;
+	private issueReporterParentWindow: BrowserWindow | null = null;
+
+	constructor(
+		@IInstantiationService private readonly instantiationService: IInstantiationService,
+		@INativeEnvironmentService private readonly environmentService: INativeEnvironmentService,
+		@ILogService private readonly logService: ILogService,
+		@IDialogService private readonly dialogService: IDialogService,
+		@INativeHostService private readonly nativeHostService: INativeHostService,
+		@IAuxiliaryWindowService private readonly auxiliaryWindowService: IAuxiliaryWindowService,
+		@IMenuService private readonly menuService: IMenuService,
+		@IContextKeyService private readonly contextKeyService: IContextKeyService,
+	) { }
+
+	//#region Used by renderer
+	async openReporter(data: IssueReporterData): Promise<void> {
+		if (data.extensionId && this.extensionIdentifierSet.has(data.extensionId)) {
+			this.currentData = data;
+			this.issueReporterWindow?.focus();
+			return;
+		}
+
+		if (this.issueReporterWindow) {
+			this.issueReporterWindow.focus();
+			return;
+		}
+		this.openAuxIssueReporter(data);
+	}
+
+	async openAuxIssueReporter(data: IssueReporterData): Promise<void> {
+		const disposables = new DisposableStore();
+
+		// Auxiliary Window
+		const auxiliaryWindow = disposables.add(await this.auxiliaryWindowService.open({ mode: AuxiliaryWindowMode.Custom, bounds: { width: 700, height: 800 } }));
+		this.issueReporterWindow = auxiliaryWindow.window;
+
+		if (auxiliaryWindow) {
+			await auxiliaryWindow.whenStylesHaveLoaded;
+			auxiliaryWindow.window.document.title = 'Issue Reporter';
+			auxiliaryWindow.window.document.body.classList.add('issue-reporter-body');
+
+			// custom issue reporter wrapper
+			const div = document.createElement('div');
+			div.classList.add('monaco-workbench');
+
+			// removes preset monaco-workbench
+			auxiliaryWindow.container.remove();
+			auxiliaryWindow.window.document.body.appendChild(div);
+			safeInnerHtml(div, BaseHtml());
+
+			// Store into config object URL
+			this.configuration = {
+				appRoot: this.environmentService.appRoot,
+				windowId: 0,
+				userEnv: {},
+				data,
+				disableExtensions: !!this.environmentService.disableExtensions,
+				os: {
+					type: '',
+					arch: '',
+					release: '',
+				},
+				product,
+				nls: {
+					// VSCODE_GLOBALS: NLS
+					messages: globalThis._VSCODE_NLS_MESSAGES,
+					language: globalThis._VSCODE_NLS_LANGUAGE
+				}
```

Comment:
I'm wondering if this isn't needed anymore since it's a Code Window now.

Reply:
yeah I'm not sure... I saw ben added these a while back

Reply:
Something worth testing. @bpasero maybe you can chime in.

Reply:
Looks like `nls` is a required type via `ISandBoxConfiguration`. 

potentially, one further step we can take is to just not use `IssueReporterWindowConfiguration extends ISandboxConfiguration` and make it it's own similarly shaped obejct?

Reply:
@justschen this kind of window configuration should not be needed at all anymore once the issue reporter goes through an auxiliary window. I am not sure if we can already remove it now, given there is a setting to go to the old issue reporter. 

An auxiliary window shares all the globals from the main window. The sandbox and window configuration is only needed for non-aux windows.